src/test/java/id/ac/itb/logistik/ditlog/DitlogAuthenticationTests.java

```diff
@@ -22,7 +21,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class DitlogUserServiceTests extends BaseTest {
+public class DitlogAuthenticationTests extends BaseTest {
```

Comment:
Since there is now roleId in the payload, the test should also check the value of that